contrib/templates/experimental/scala-job/template/databricks.yml.tmpl

```diff
@@ -0,0 +1,52 @@
+# This is a Databricks asset bundle definition for {{.project_name}}.
+# See https://docs.databricks.com/dev-tools/bundles/index.html for documentation.
+bundle:
+  name: {{.project_name}}
+
+workspace:
```

Comment:
Could you follow the exact same conventions as seen in templates like https://github.com/databricks/cli/blob/874a05a27b750f2f732436d2b43d90997dc6b32c/libs/template/templates/default-python/template/%7B%7B.project_name%7D%7D/databricks.yml.tmpl#L10-L31, where there are `dev` and `prod` targets, and the workspace host is part of the targets section?